spyder/plugins/projects/plugin.py

```diff
@@ -195,15 +195,13 @@ def setup_menu_actions(self):
             for project in self.recent_projects:
                 if self.is_valid_project(project):
                     name = project.replace(get_home_dir(), '~')
-
-                    def slot():
-                        self.switch_to_plugin()

```

Comment:
We do not need to call this here, since it is the first thing that is called in `open_project`